AIAgent/common/game.py

```diff
@@ -89,12 +90,12 @@ class MoveReward:
     ForVisitedInstructions: int
 
     def __eq__(self, __o) -> bool:
-        if type(self) != type(__o):
+        if type(self) != type(__o):  # noqa: E721
```

Comment:
Why the problem can not be fixed, but should be ignored?

Reply:
Solved